malariagen_data/ag3.py

```diff
@@ -13,13 +14,17 @@
 
 import malariagen_data  # used for .__version__
 
+from .anoph.base import base_params
 from .anopheles import AnophelesDataResource, gplt_params
 from .util import (
     DIM_SAMPLE,
     DIM_VARIANT,
     Region,
     da_from_zarr,
     init_zarr_store,
+    parse_multi_region,
+    parse_single_region,
```

Comment:
The functions for parsing `region` parameter have been reworked to make sure it's clear when a function will accept multiple regions in a list, versus when only a single region is accepted.